lib/rpc/SingleJsonRpcProvider.ts

```diff
@@ -360,46 +365,40 @@ export class SingleJsonRpcProvider extends StaticJsonRpcProvider {
 
   ///////////////////// Begin of override functions /////////////////////
 
-  override getBlockNumber(callType = CallType.NORMAL): Promise<number> {
-    return this.wrappedFunctionCall(callType, 'getBlockNumber', this._getBlockNumber.bind(this))
+  override getBlockNumber(): Promise<number> {
+    return this.wrappedFunctionCall(CallType.NORMAL, 'getBlockNumber', this._getBlockNumber.bind(this))
```

Comment:
why did we change this to always normal?

Reply:
Because for `HEALTH_CHECK` (in `getBlockNumber_EvaluateHealthiness`) and `LATENCY_EVALUATION` calls (in `getBlockNumber_EvaluateLatency`), we will use `this._getBlockNumber` instead of `getBlockNumber`. So this `getBlockNumber`'s only use case is to make normal calls.